server/app.js

```diff
@@ -32,6 +32,18 @@ const csrfProtection = csrf({ cookie: true })
 const parseForm = bodyParser.urlencoded({ limit: '50mb', extended: true })
 const app = express();
 
+if(process.env.NODE_ENV === 'development') {
+  const livereload = require('livereload')
+  const connectLiveReload = require('connect-livereload')
```

Comment:
should these require statements move to the top of the file? I'm not used to conditional requires and I think it won't cause any harm to require them if we only use them in the dev environment.

Reply:
[f67f383](https://github.com/AMP-SCZ/dpdash/commit/f67f383401bf0105196fd976a7c6a4fce63d6f22)
👍